### ISSUE TITLE
Broken develop docs

### ISSUE THREAD

**Issue by kain88-de** [MDAnalysis/mdanalysis#1265]
https://travis-ci.org/MDAnalysis/mdanalysis/jobs/215530626#L1089

Looks like my recent changes to our travis build have broken the doc deployment. I don't understand the error. If I execute the commands like that on my machine I get the folder.

**Comment by orbeckst** [comment]
More explicit than your link above. Deployment in the merge-to-develop failed in [Job #3187.3](https://travis-ci.org/MDAnalysis/mdanalysis/jobs/215714720) because one parameter is not defined any more (`$2`).:
```
/home/travis/build/MDAnalysis/mdanalysis/maintainer/deploy_docs.sh: line 36: cd: package/doc/html: No such file or directory

/home/travis/build/MDAnalysis/mdanalysis/maintainer/deploy_docs.sh: line 24: $2: unbound variable
```

(I think also relevant for PR #1266.)

**Comment by kain88-de** [comment]
The unbound variable is a problem of the custom exit function that is called when a environment variable isn't found.

**Comment by jbarnoud** [comment]
This issue seems fixed. No?

Now we need to get rid of all the errors and warnings so we can have the build fail on warnings.

**Comment by kain88-de** [comment]
Yes you just need to activate the warnings to error conversion in the setup.cfg file.

**Comment by jbarnoud** [comment]
Yes, but if we activate it now, it will fail. So we have to fix the existing warnings.

**Comment by orbeckst** [comment]
The [devdocs](http://www.mdanalysis.org/mdanalysis/) are fine again: this got fixed in PR #1266.
